### PR TITLE
Add missing npm dep: load-grunt-tasks

### PR DESCRIPTION
**Pull request by henrahmagix**
And other improvements: see commits

@incuna/frontend Please merge, ta!

### CODE REVIEW

**Comment by Minglee01** [comment]
looks ok

File: Gruntfile.js [hunk 1]
```diff
@@ -8,7 +8,13 @@ module.exports = function (grunt) {
 
     if (grunt.option('help')) {
         // Load all tasks so they can be viewed in the help: grunt -h or --help.
-        require('load-grunt-tasks')(grunt);
```
Comment (Minglee01) [inline Gruntfile.js:11]: what was the error?

**Comment by henrahmagix** [comment]
Without
image


With
image

**Comment by henrahmagix** [comment]
`load-grunt-tasks` default pattern copied from https://github.com/sindresorhus/load-grunt-tasks#load-all-grunt-tasks: was required so `'!grunt-incuna-plugins'` could be included

**Comment by henrahmagix** [comment]
Thanks!

**Pull request merged**